src/corporacreator/preprocessors/fr.py

```diff
@@ -8,5 +43,9 @@ def fr(client_id, sentence):
     Returns:
       (str): Cleaned up sentence. Returning None or a `str` of whitespace flags the sentence as invalid.
     """
-    # TODO: Clean up fr data
-    return sentence
+    text = maybe_normalize(sentence, mapping=FR_NORMALIZATIONS + SPELLED_ACRONYMS)
+    text = replace_numbers(text, locale='fr', ordinal_regex=FIND_ORDINAL_REG)
+    text = text.replace('’', "'").replace('\u00A0', ' ')
```

Comment:
Hm, shouldn't we keep UTF-8 quotes ?

Reply:
since both STT and TTS will interpret them in the same, what is the point of having a distinction between `'` and `’` ?

Reply:
Just that `’` is the actual french one :-)

Reply:
> Just that `’` is the actual french one :-)

IMHO, except from official documents, `'` is more used than `’`. Since it do not change anything from a technical perspective a suggest to stick to `'`

Reply:
fixed, see https://github.com/mozilla/CorporaCreator/pull/87/commits/934ec386a2334dabaef9746ec81d536a0f848be7